hellonode/server.js

```diff
@@ -15,11 +15,19 @@
  */
 
 var http = require('http');
+var os = require('os');
+
```

Comment:
it might be nice to declare the port here, and give an option to declare it via env_var

```js
const port = process.env.NODE_PORT || 8080;
```